src/components/global/button.tsx

```diff
@@ -3,12 +3,17 @@
 interface ButtonProps {
   children: string | number;
   onClick: () => void;
+  variant?: "default" | "outline";
 }
 
-const Button = ({ children, onClick }: ButtonProps) => {
+const Button = ({ children, onClick, variant = "default" }: ButtonProps) => {
   return (
     <button
-      className="rounded-md bg-starlight-blue px-8 py-2 font-serif text-white hover:opacity-80"
+      className={`rounded-md px-8 py-2 font-serif hover:opacity-80 ${
+        variant === "outline"
```

Comment:
outline wont be the only variant we have, so it should be able to handle multiple variants nicely